pkg/operator/configobservation/util/util.go

```diff
@@ -0,0 +1,29 @@
+package util
+
+import (
+	appsv1 "k8s.io/api/apps/v1"
+
+	operatorapi "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+// InjectObservedProxyInDeploymentContainers takes an observed proxy config and returns a patched Deployment with proxy env vars set.
+func InjectObservedProxyInDeploymentContainers(deployment *appsv1.Deployment, opSpec *operatorapi.OperatorSpec, observedConfig []byte, fields ...string) (*appsv1.Deployment, error) {
```

Comment:
`observedConfig` and `fields` are not used anywhere

Reply:
removed